(docs) Add release notes for 7.0.0 (#3373)
puppetlabs/puppetdb: documentation/release_notes/release_notes_latest.markdown

---

## PuppetDB 7.0.0

Released 19 November 2020

### Bug fixes

- PuppetDB no longer retries queries internally, suppressing some transient

>   connection errors. Instead, it will return an error code more immediately.

Reviewer: ```suggestion
  connection errors. Instead, it immediately returns an error code.
```

>   For the moment, the previous behavior can be restored by setting the

Reviewer: ```suggestion
You can restore the previous behaviour by setting the
```

  `PDB_USE_DEPRECATED_QUERY_STREAMING_METHOD` environment variable. See the
  [configuration information](https://puppet.com/docs/puppetdb/latest/configure.html#experimental-environment-variables)
  for further details.

- PuppetDB won't hold an extra database connection open while generating query
  responses. Previously it would create and hold an extra connection open during

>   the initial phase of the response. For the moment, the previous behavior can

Reviewer: ```suggestion
  the initial phase of the response. You can restore the previous behavior
```

>   be restored by setting the `PDB_USE_DEPRECATED_QUERY_STREAMING_METHOD` environment

Reviewer: ```suggestion
by setting the `PDB_USE_DEPRECATED_QUERY_STREAMING_METHOD` environment
```
